docs/semgrep-supply-chain/overview.md

```diff
@@ -27,21 +27,23 @@ Object.entries(frontMatter).filter(
 
 <AdmonitionSscLicense />
 
-![Semgrep Supply chain overview page](/img/sc-overview.png)
-_Figure 1_. Semgrep Supply Chain overview page.
+![Semgrep Supply chain Vulnerabilities page](/img/sc-vulns.png)
+_Figure 1_. Semgrep Supply Chain Vulnerabilities page.
 
 Semgrep Supply Chain parses lockfiles for a list of dependencies, then scans your codebase using **rules **written with Semgrep's pattern syntax. Supply Chain rules specify the following:
 
 * The range of versions that contain the dependency's vulnerability.
 * A pattern for vulnerable code, such as passing in unsanitized data.
 * The severity of the vulnerability.
 
-Semgrep Supply Chain generates a **finding **when it detects a match. If the dependency's version is within the range and finds the matching code within your codebase, the finding is **reachable**.
+Semgrep Supply Chain generates a **finding** when it detects a match. If the dependency's version is within the range and finds the matching code within your codebase, the finding is **reachable**.
 
 A finding is **unreachable** if the dependency contains a known vulnerability, but the vulnerable matching code is not used in your codebase.
 
 In Semgrep Cloud Platform, specific findings of a dependency and code match are called **usages**. Usages are grouped by their **vulnerability**. Vulnerabilities in Semgrep Supply Chain typically have a CVE number corresponding to the record in the [CVE Program](https://www.cve.org/About/Overview).
 
+Semgrep Cloud Platform also includes a list of **Advisories** for reference. Advisories include all vulnerabilities covered by Semgrep Supply Chain, regardless of whether the related dependency is used in scanned code.
```

Comment:
"Semgrep Cloud Platform also includes a list of **Advisories**, a database of all disclosed CVEs for open source dependencies in ecosystems covered by Semgrep Supply Chain. This list is independent of whether the related dependency is used in scanned code (See Vulnerabilities)"

Reply:
we have a script that automatically generates parity rules as soon as GHSA discloses something new--the writing part only applies to reachability rules

Reply:
Discussed with Andy and made into a new Linear task: https://linear.app/r2c/issue/MKT-601/clarity-on-advisories-transitivity-how-we-work

Resolving.